exec/java-exec/src/main/java/org/apache/drill/exec/record/TypedFieldId.java

```diff
@@ -241,7 +241,7 @@ public TypedFieldId build() {
         secondaryFinal = finalType;
       }
 
-      MajorType actualFinalType = finalType;
+      //MajorType actualFinalType = finalType;
```

Comment:
Should we keep this commented code?

Reply:
Not sure. Not sure what the comments here are about, so I thought I'd leave them. Just commenting out an unused variable.